saleor/dashboard/graphql/product/mutations.py

```diff
@@ -42,3 +44,23 @@ class CategoryDelete(ModelDeleteMutation):
 
     class Meta:
         model = models.Category
+
+
+class ProductCreateMutation(ModelFormMutation):
+    class Arguments:
+        product_type = graphene.ID
```

Comment:
In few other queries we already used the suffix `_id` for such fields, so maybe we should also rename these two to `product_type_id` and `category_id` to make it consistent.